server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestGetScriptLanguageAction.java

```diff
@@ -13,13 +13,16 @@
 import org.elasticsearch.client.internal.node.NodeClient;
 import org.elasticsearch.rest.BaseRestHandler;
 import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.Scope;
+import org.elasticsearch.rest.ServerlessScope;
 import org.elasticsearch.rest.action.RestToXContentListener;
 
 import java.io.IOException;
 import java.util.List;
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
+@ServerlessScope(Scope.PUBLIC)
```

Comment:
Can you please remove this one from the PR? We are still in discussions about whether it is necessary (and there are actually 2 versions of this API under different paths).